examples/simple.rs

```diff
@@ -66,23 +64,20 @@ fn handle_packets(
     mut reader: EventReader<NetworkEvent>,
 ) {
     for event in reader.iter() {
-        match event {
```

Comment:
This code serves as an example how to approach reading events. Usually we are interested in processing all types of events, not just one specific (`if let Specific(`), so rewriting it as `if let` counters educational value.

Reply:
I see. This is the same problem as the one with the "compressed" if.